static/app/views/onboarding/targetedOnboarding/onboarding.tsx

```diff
@@ -192,7 +203,7 @@ function Onboarding(props: Props) {
               <stepObj.Component
                 active
                 stepIndex={stepIndex}
-                onComplete={() => goNextStep(stepObj)}
+                onComplete={() => stepObj && goNextStep(stepObj)}
```

Comment:
@Neo-Zhixing If `stepObj` is falsy, wouldn't `stepObj.Component` just error out anyways?

Reply:
Yeah so this is just there to make TypeScript happy.